lib/byron/src/Cardano/Wallet/Byron/Network.hs

```diff
@@ -274,15 +284,21 @@ newNetworkLayer tr bp addrInfo versionData = NetworkLayer
 -- callback.
 --
 -- See also 'send' for invoking commands.
-data NetworkClientCmd (m :: * -> *)
+data ChainSyncCmd (m :: * -> *)
     = CmdFindIntersection
         [Point ByronBlock]
         (Maybe (Point ByronBlock) -> m ())
     | CmdNextBlocks
-        (NextBlocksResult (m Byron) ByronBlock -> m ())
+        (NextBlocksResult (Point ByronBlock) ByronBlock -> m ())
     | CmdCurrentNodeTip
         (Tip ByronBlock -> m ())
 
+-- | Sending command to the localTxSubmission client. See also 'ChainSyncCmd'.
+data LocalTxSubmissionCmd (m :: * -> *)
+    = CmdSubmitTx
+        (GenTx ByronBlock)
+        (Maybe String -> m ())
```

Comment:
`GenTx ByronBlock` is basically a `Tx` corresponding to the `ByronBlock`?

`Maybe String` -> is `Nothing` success and `Just str` an error?

Reply:
`Gen` stands for _Generalized_  as far as I understood, and yes, it corresponds to classic transactions from byron blocks although, there are several constructors for a `GenTx`, some of them being used to construct delegation transaction for instance. 

And yes, the way the `LocalTxSubmission` protocol works is by returning a `Maybe err` as a message, where `err` is currently instantiated as a `String`.